File: Retangulo.py
class Retangulo(object):

    #construtor
    def __init__(self, base, altura):
        self.basE = base
        self.alturA = altura
    #metodo area
    def area(self):
        areA = self.basE * self.alturA
        print(areA)
    #metodo perimetro
    def perimetro(self):
        perimetrO = 2*(self.basE + self.alturA)
        print(perimetrO)

File: 20_08/20_08/Retangulo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _20_08
{
    class Retangulo
    {
        //atributos
        private int basE;
        private int alturA;

        //metodos
        //construtor - metodo que cria objetos da classe
        public Retangulo()
        {
            basE = 10;
            alturA = 5;
        }
        public void imprime()
        {
            Console.WriteLine("Base = " + basE + "\nAltura=" + alturA + "\n");
        }
        public void setAltura(int a)
        {
            alturA = a;
        }
        
    }
}
